docs/pages/docs/guides/huggingface.mdx

```diff
@@ -232,6 +236,23 @@ export default function Completion() {
 
 </Steps>
 
+## Guide: Using Production-ready Inference Endpoints
+
+Inference Endpoints offers a secure solution to deploy models from the Hub on dedicated and autoscaling infrastructured. It supports popular libraries such as `transformers` and `diffusers` out-of-the-box, but users can also implement custom handlers for their own solutions. Read more about Inference Endpoints in [their documentation](https://huggingface.co/docs/inference-endpoints/index).
+ With Inference Endpoints, you can even use state of the art open-source LLMs requiring more compute such as [Falcon 40B](https://huggingface.co/blog/inference-endpoints-llm).
```

Comment:
```suggestion
 With Inference Endpoints, you can even use state-of-the-art open-source LLMs requiring more compute such as [Falcon 40B](https://huggingface.co/blog/inference-endpoints-llm).
```

Lots of hypens, but I think it's supposed to be hyphenated when using as a compound adjective.